pom.xml

```diff
@@ -267,8 +267,6 @@
     <version.org.springframework>5.3.34</version.org.springframework>
     <version.org.springframework.boot>2.7.18</version.org.springframework.boot>
     <version.org.springframework.osgi>1.2.1</version.org.springframework.osgi>
-    <!-- spring-security.version and tomcat.version overrides version from spring-boot-dependencies bom -->
-    <tomcat.version>9.0.86</tomcat.version>
```

Comment:
it is required for the comment above, what is the reason to remove it? I don't see the problem even i would really see as a problem to removed and keep not overwriting it in spring bom